src/cqparts/display/cqparts_server.py

```diff
@@ -1,21 +1,23 @@
-# generate the files and notify the cqparts server
-# look at https://github.com/zignig/cqparts-server
-# copied and edited from web.py
-# Copyright 2018 Peter Boin
-# and Simon Kirkby 2018
+""" generate the files and notify the cqparts server
+ look at https://github.com/zignig/cqparts-server
+ copied and edited from web.py
+ Copyright 2018 Peter Boin
+ and Simon Kirkby 2018
+"""
 
 import os
-import sys
-import inspect
 import time
-import requests
 import tempfile
+import shutil
 
 import logging
-log = logging.getLogger(__name__)
+import requests
 
 from .environment import map_environment, DisplayEnvironment
 
+LOG = logging.getLogger(__name__)
```

Comment:
`log` is in conventionally in lower-case (at least, it is for `cqparts` and `cadquery`)